plugins/scaffolder/src/components/TemplateEditorPage/TemplateEditorForm.tsx

```diff
@@ -212,11 +212,17 @@ export function TemplateEditorFormDirectoryEditorDryRun(
       return;
     }
 
-    await dryRun.execute({
-      templateContent: selectedFile.content,
-      values: data,
-      files: directoryEditor.files,
-    });
+    try {
+      await dryRun.execute({
+        templateContent: selectedFile.content,
+        values: data,
+        files: directoryEditor.files,
+      });
+      setErrorText();
+    } catch (e) {
+      setErrorText(String(e.cause || e));
```

Comment:
hm i wish stringifyError did the right thing here including the cause, so we could use it

Reply:
Perhaps, this shows the top-level error message in the alert and the cause in the text box